src/thermobench.cpp

```diff
@@ -203,6 +204,35 @@ void write_arr_csv(FILE *fp, double *arr, int size)
     fprintf(fp, "\n");
 }
 
+void wait_cooldown(char* exec_dir){
+
+    FILE *fp = fopen(state.sensors[0].path, "r");
```

Comment:
I'd like to have a function read_sensor(), which would be used in this function as well as in `measure_timer_cb()`.